packages/story-editor/src/components/panels/document/taxonomies/HierarchicalTermSelector.js

```diff
@@ -71,6 +77,8 @@ const ButtonContainer = styled.div`
 const LinkButton = styled(Button).attrs({
   variant: BUTTON_VARIANTS.LINK,
 })`
+  ${(props) => props._showAddNewCategory && 'display: none;'}
```

Comment:
nit: Let's use a [transient prop](https://styled-components.com/docs/api#transient-props) here instead 
```suggestion
  display: ${({$isVisible}) => $isVisible ? 'none' : 'block')};
```

Reply:
For sure!